<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchExportDialog</class>
 <widget class="QDialog" name="BatchExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HOCR Batch Export</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInput">
      <attribute name="title">
       <string>Input files</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0" colspan="3">
        <widget class="QWidget" name="widgetFiles" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QTreeView" name="treeViewInput">
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTreeView" name="treeViewOutput">
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelInput">
            <property name="text">
             <string>Input:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelOutput">
            <property name="text">
             <string>Output preview:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelExportLevel">
         <property name="text">
          <string>Group outputs of N lowest levels:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelSourceFolder">
         <property name="text">
          <string>Select source folder:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditSourceFolder">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="toolButtonSourceFolder">
         <property name="icon">
          <iconset theme="folder-open">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelFormat">
         <property name="text">
          <string>Format:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxFormat"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QSpinBox" name="spinBoxExportLevel"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOptions">
      <attribute name="title">
       <string>Export options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
